packages/components/Molecules/Form/fields/fieldRplhopo.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <rpl-input
+    v-model="value"
+    :disabled="false"
+    placeholder=""
+    :readonly="false"
+    :identifier="getFieldID(schema)"
+    autocomplete="off"
+    tabindex="-1"
+  />
+</template>
+
+<script>
+import RplInput from '../Input.vue'
+import { abstractField } from 'vue-form-generator'
+export default {
+  name: 'RplHopo',
+  components: {
+    RplInput
+  },
+  mixins: [abstractField],
+  mounted () {
+    this.schema.label ??= 'Important email'
+    this.schema.model ??= 'honeypot'
```

Comment:
Just to be safe, has this been tested on IE11 to make sure the `??=` operator is being properly transpiled? If it has been tested and works, then all good 👍

Reply:
@alan-cole I will check browser support.

Reply:
Not compatible, changed.